content/courses/onchain-development/anchor-pdas.md

```diff
@@ -544,7 +544,7 @@ pub mod anchor_movie_review_program{
         description: String,
         rating: u8,
     ) -> Result<()> {
-	// We require that the rating is between 1 and 5
+    // We require that the rating is between 1 and 5
```

Comment:
Honestly these comments can just be deleted. Comments should be for the 'why' - anyone who reads this code will know what it does.